benchmarks/multimodal/local_media_server/main.py

```
@@ -0,0 +1,44 @@
+# limitations under the License.
+
+import requests
+
+from .args import parse_args, parse_images
+from .local_media_server import run_server
+
+
+def download(images: dict[str, str]) -> dict[str, bytes]:
+    downloaded: dict[str, bytes] = {}
+    for name, url in images.items():
+        response = requests.get(url)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)